service/src/main/kotlin/com/mashup/dojo/service/QuestionService.kt

```diff
@@ -0,0 +1,42 @@
+package com.mashup.dojo.service
+
+import com.mashup.dojo.domain.Question
+import com.mashup.dojo.domain.QuestionId
+import com.mashup.dojo.domain.QuestionType
+import io.github.oshai.kotlinlogging.KotlinLogging
+import org.springframework.stereotype.Service
+import org.springframework.transaction.annotation.Transactional
+import java.time.LocalDateTime
+
+private val log = KotlinLogging.logger {}
+
+@Service
+@Transactional(readOnly = true)
+class QuestionService {
+    @Transactional
+    fun createQuestion(
+        content: String,
+        type: QuestionType,
+        imageUrl: String,
+    ): Question {
+        // todo create questionEntity
+        // return QuestionRepository.save(content, type, imageUrl).toQuestion
+        val question = sampleQuestion
+
+        log.info { "Create Question Success : $question" }
+
+        return sampleQuestion
+    }
+
+    companion object {
+        val sampleQuestion =
```

Comment:
```suggestion
    companion object {
        val SAMPLE_QUESTION =
```
상수니까 대문자로다가 멋지게 하면 어떱니깡